renderers/server-render.jsx

```diff
@@ -0,0 +1,172 @@
+import Rx from 'rxjs';
```

Comment:
As far as directory structure goes, since this is the server entry point, I personally find it a little strange/confusing that it lives in this sub-directory alongside browser-render.jsx. 

Could this file just live top level, and maybe move `browser-render` until `utils/` or somewhere else?